server/src/integration/java/org/opentosca/toscana/core/BaseJUnitIntegrationTest.java

```diff
@@ -0,0 +1,9 @@
+package org.opentosca.toscana.core;
+
+import org.opentosca.toscana.IntegrationTest;
+
+import org.junit.experimental.categories.Category;
+
+@Category(IntegrationTest.class)
+public abstract class BaseJUnitIntegrationTest extends BaseJUnitTest {
```

Comment:
What the heck is a BaseJUnitIntegrationTest? :grin:
A test is either an integration or an unit test; never a UnitIntegrationTest. I am confused. Can you explain?

Reply:
Well it represents a Integration test that does not need a spring context. Didn't think of a better name. But maybe `BaseIntegrationTest` is better

Reply:
Yes! Maybe also rename BaseJunitTest to BaseUnitTest.

Reply:
Yeah technically BaseJUnitIntegrationTest is not a contradiction; but then `JUnit` information is quite unnecessary..